backend/ttrs/tests.py

```diff
@@ -429,3 +429,13 @@ def test_destroy(self):
     def test_receive(self):
         response = self.get_test('/ttrs/received-time-tables/3/receive/')
         self.assertIsNotNone(response.json()['received_at'])
+
+
+class RecommendViewTest(BaseTestCase):
+    def setUp(self):
+        super().setUp()
+        # self.sign_in()
+
+    def test_retrieve(self):
+        response = self.get_test('/ttrs/recommends/')
+        response = self.get_test('/ttrs/recommends/?year=2018&semester=1학기')
```

Comment:
Please append a blank line at the end of the file.

Reply:
Sure.